pkg/generators/router/router_test.go

```diff
@@ -115,39 +164,39 @@ func TestCreateTemplateCtx(t *testing.T) {
 			Version:     "Version",
 		},
 		Paths: map[string]path{
-			"/some/path": path{
+			"/some/path": {
 				GET: &endpoint{
 					Summary:     "GET Summary",
 					Description: "GET Description",
-					OperationID: "GETOperationID",
+					OperationID: "GETOperationId",
```

Comment:
Is there a specific reason to change the casing here?

Reply:
Yes, `toPascalCase` changes `ID` to `Id` so, now all the test data must be adjusted.